requirements.in

```diff
@@ -7,7 +7,7 @@ Flask-WTF==1.0.1
 # Should be pinned until a new gunicorn release greater than 20.1.0 comes out. (Due to eventlet v0.33 compatibility issues)
 git+https://github.com/benoitc/gunicorn.git@1299ea9e967a61ae2edebe191082fd169b864c64#egg=gunicorn[eventlet]==20.1.0
```

Comment:
unrelated to this but i just saw this and got very sad because gunicorn STILL haven't had a mainline release in the last two years

Reply:
😢